misc/tb/src/TB.java

```diff
@@ -86,6 +86,11 @@ public static void main(String[] args) throws Exception {
         config.setProperty("database.password", ns.getString("password"));
         config.setProperty("database.server.name", "tb");
         config.setProperty("database.history", "io.debezium.relational.history.FileDatabaseHistory");
+
+        config.setProperty("database.allowPublicKeyRetrieval", "true");
+        config.setProperty("log_bin", "mysql-bin");
+        config.setProperty("binlog_format", "row");
```

Comment:
I'm a bit confused. Aren't `log_bin` and `binlog_format` MySQL system configuration parameters that need to be set in `my.cnf`, not in Debezium connection parameters?

Reply:
Sorry for that - initially I just wanted to make it work locally and did not think about publishing changes